base/applications/cmdutils/more/more.c

```diff
@@ -456,7 +761,9 @@ LoadRegistrySettings(HKEY hKeyRoot)
      * Buffer big enough to hold the string L"4294967295",
      * corresponding to the literal 0xFFFFFFFF (MAXULONG) in decimal.
      */
-    DWORD Buffer[6];
+    WCHAR Buffer[11];
+    C_ASSERT(sizeof(Buffer) >= sizeof(L"4294967295"));
```

Comment:
The following might be better:
```suggestion
    WCHAR Buffer[sizeof("4294967295")];
```
It automatically ensures the correct size (the sizeof will also count the null-terminator) and makes the intent more concise.

Reply:
OK.